{{ cookiecutter.package_name }}/setup.cfg

```diff
@@ -36,7 +36,7 @@ select = E101,W191,W291,W292,W293,W391,E111,E112,E113,E901,E902
 exclude = extern,sphinx,*parsetab.py
 
 [metadata]
-package_name = {{ cookiecutter.package_name }}
+package_name = {{ cookiecutter.module_name }}
```

Comment:
it's still called ``{{ cookiecutter.package_name }}`` above in the filename, what is the reason of changing it here?